index.bs

```diff
@@ -967,6 +970,39 @@ A string or dictionary identifying a cryptographic algorithm and optionally a se
 defined in [[!WebCryptoAPI]].
 
 
+## Algorithm to Relax the Same-Origin Restriction ## {#algo-relax-same-origin-restriction}
+
+In general, the same-origin policy prevents web applications from interacting
+within user agents, unless they have the same origin. However, there are many
+specific facets and exceptions to this overall policy.
+
+For example, cookies [[RFC6265]] have same-origin restrictions similar to those
+of the HTML `document.domain` attribute. Both permit <a>relaxing the same-origin
+restriction</a> in a particular fashion, under certain circumstances.
```

Comment:
And this one should probably say something more like:

> We want to allow an rpid to represent a parent domain, subject to constraints similar to those of cookies or `document.domain`.

or something like that.